service/storage_proxy.cc

```diff
@@ -596,6 +596,8 @@ class storage_proxy::remote {
                     } else if constexpr(std::is_same_v<Ex, replica::stale_topology_exception>) {
                         msg = e.what();
                         return error::FAILURE;
+                    } else if constexpr (std::is_same_v<Ex, replica::abort_requested_exception>) {
+                        return error::FAILURE;
```

Comment:
I think it could make sense to assign the error message to the `msg` variable. This will cause the "aborted requested" error message to be propagated back to the user, giving them a bit more context.